monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export function RenderMachine(val){
+    return (
+      <span>{val.ip_addr} {(val.domain_name ? " (".concat(val.domain_name, ")") : "")}</span>
+    )
+}
+
+export function renderMachineFromSystemData(data) {
```

Comment:
maybe rename data to machine

Reply:
This function parses system info collector data to create a string representation for machine (same as render machine, just parses different format of data).